pyproject.toml

```
@@ -21,7 +21,7 @@ exclude = '''
 
 [tool.poetry]
 name = "scale-nucleus"
-version = "0.3.1"
+version = "0.4.0"
```

we just incremented this for Model CI 😢 
```suggestion
version = "0.4.1"
```